docs/README.md

```diff
@@ -99,12 +86,16 @@ The second example belongs to the measure variable of a dataset where the measur
 We expect all values in this dataset to be either "1" or "2", as this dataset only considers "Male" or "Female". But we also expect a code "0" to be present in the dataset, where it represents "Unknown". A row with "0" as measure is therefore not considered invalid. A value domain with a code list like this is what we would call an __enumerated value domain.
 
 
-### UNIT TYPE
-Description of the unit the data describes. A "fødselsnummer" can be used to identify a PERSON. Another example would be an "organisasjonsnummer" that would be used to identify a "FORETAK".
-* **shortName**: Machine readable name.
-* **name**: Human readable name.
-* **description**: Description of the unit type.
-
+### UNIT TYPES
+* **PERSON**: Representation of a person in the microdata platform. Columns with this unit type should contain FNR.
```

Comment:
Maybe translate all types to English in the descriptions? 🤔

Reply:
We can attempt this in the future once we get the unitTypes with languagefields for both Norwegian and English 😅 I have no idea what to translate FORETAK/VIRKSOMHET to according to the definition used in the platform f.ex.